### ISSUE TITLE
Redesign Nautilus Pathbar

### ISSUE THREAD

**Issue by NicoHood** [NicoHood/arc-theme#133]
See https://github.com/NicoHood/arc-theme/issues/124 and https://github.com/NicoHood/arc-theme/pull/132

The nautilus pathbar design should be based on the default gnome theme.

**Comment by jnsh** [comment]
Here a basic idea for the design, never mind that the colors are a little off.
![screenshot from 2018-10-08 03-33-52](https://user-images.githubusercontent.com/39574454/46588911-280e1f80-caac-11e8-85c8-1b14391edcae.png)
![screenshot from 2018-10-08 03-35-04](https://user-images.githubusercontent.com/39574454/46588910-27758900-caac-11e8-868b-3fdb3adfa3a8.png)


I've got the css done, and can work it to sass for a PR if this looks ok. That would also fix #139.

The "underlined" buttons are hovered with mouse cursor. This highlight could possibly be improved, but I couldn't come up with better ideas for now. Basically it follows the Adwaita design, apart from using the highlight color. However, the `path-bar-box` height is slightly lower in Arc, and the "underline" may be a bit too close to the text. Increasing the `path-bar-box` height isn't a good solution, since the height should be the same as other headerbar buttons.

If anyone has ideas for that, or any other suggestions or feedback, let me know.

Here's the css I used if anyone wants to play around with it in `~/.config/gtk-3.0/gtk.css`:
```
.nautilus-window .path-bar-box {
  border-radius: 3px;
}

.nautilus-window .path-bar > box > button {
  background-color: transparent;
  border-color: transparent;
  border-radius: 0;
  margin: 0 2px;
  padding: 1px;
}

.nautilus-window .path-bar > box > button:hover {
  box-shadow: 0 -3px rgb(99, 99, 200) inset;
}
```
